src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
+   abstract class Machine {
+       public abstract void doWork();
+       public abstract void stopWork();
+   }
```

Comment:
The abstract class `Machine` should be in a separate file. This will make the code more organized and adhere to the single responsibility principle.